packages/modules/nats/package.json

```diff
@@ -28,10 +28,9 @@
     "prepack": "shx cp ../../../README.md . && shx cp ../../../LICENSE .",
     "build": "tsc --project tsconfig.build.json"
   },
-  "devDependencies": {
-    "nats": "^2.28.0"
-  },
   "dependencies": {
+    "@nats-io/jetstream": "^3.0.2",
```

Comment:
Both of these should be devDependencies rather than dependencies.

`npm i -D ...`